core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/named/AddNamedOperationHandler.java

```diff
@@ -41,15 +41,23 @@
  */
 public class AddNamedOperationHandler implements AddToCacheHandler<AddNamedOperation> {
 
+    public static final Boolean DEFAULT_IS_NESTED_NAMED_OPERATIONS_ALLOWED = false;
```

Comment:
The only reference to this is from `StoreProperties.java` which is in a different package. This field should go there instead.

Reply:
Yeah, I know what you mean. If you was interested in the default values and changing it for nested AddNamedOperation, where would you start to look? I'd go to AddNamedOperationHandler and start there. Not sure I'd dive into Properties.
Thoughts?